introduction/architecture.md

```diff
@@ -111,6 +110,68 @@ while maintaining support for multiple indexes.
 For more on the motivation and design of TimescaleDB's adaptive space/time
 chunking, please see our [technical blog post][chunking].
```

Comment:
Cut this reference to adaptive chunking giving that current implementation has been deprecated for some time now.

Reply:
Ok, gonna delete this in a different PR to master branch